### DISCUSSION TITLE
两个功能小建议

### DISCUSSION THREAD

**Discussion by YINGZILOU** [maicss/chrome-shanbay-v2#10] [Ideas]
1. 查词后加入生词本时，连生词所在的句子一起加入例句，这样复习生词时比较有效。
2. 点击工具栏的扇贝图标后出来的小面板不太好看哦，点击后如果能出现类似有道词典插件的文本搜索框，能简单中英查词就更好啦，然后在这个小面板里放入目前的三个功能按钮。

**Comment by maicss** [comment]
1，这个应用完全依赖于扇贝的能力，扇贝好像没这个能力吧
2，这个能力工作量有点不小哎，我暂时应该没时间做，以后看看吧

**Comment by YINGZILOU** [comment]
哦，这样的啊，开始以为是扇贝做的插件^_^
不管怎样，还是谢谢你开发好用的免费插件，回复还这么及时。

**Reply by maicss** [reply]
You're welcome.
